peef/peef.go

```diff
@@ -122,7 +127,25 @@ func getSymbolCurrentPrice(symbol string, key string) float32 {
 		log.Fatal(err)
 	}
 
-	fmt.Println(prices)
+	return prices[0]
+}
+
+func buildPriceUrl(symbol string, key string) string {
+	// TODO: see if base can be set using NewRequest
+
+	baseUrl := fmt.Sprintf("https://financialmodelingprep.com/api/v3/quote-short/%s", symbol)
```

Comment:
Noticed that the `.Add()` in the example was just adding query params. The *symbol* is an endpoint I believe, so I wanna see what else `NewRequest` can do,